src/libs/TransactionRecoveryApi/AxelarGMPRecoveryAPI.ts

```diff
@@ -476,6 +475,19 @@ export class AxelarGMPRecoveryAPI extends AxelarRecoveryApi {
     );
     return txStatus?.status === GMPStatus.DEST_EXECUTED;
   }
+  /**
+   * Check if given transaction is already confirmed.
+   * @param txHash string - transaction hash
+   * @returns Promise<boolean> - true if transaction is already confirmed
+   */
+  public async isConfirmed(txHash: string): Promise<boolean> {
+    const txStatus: GMPStatusResponse | undefined = await this.queryTransactionStatus(txHash).catch(
+      () => undefined
+    );
+    return [GMPStatus.SRC_GATEWAY_CONFIRMED, GMPStatus.DEST_GATEWAY_APPROVED].includes(
```

Comment:
what does `SRC_GATEWAY_CONFIRMED` mean?

Reply:
status is `confirmed`. but we translate that to the use (via `parseGMPStatus`) to be more descriptive to read `source_gateway_confirmed` instead.